packages/@aws-cdk/aws-amplify-alpha/test/integ.app-asset-deployment.ts

```diff
@@ -1,25 +1,28 @@
-// import * as path from 'path';
-// import { Asset } from '@aws-cdk/aws-s3-assets';
+import * as path from 'path';
 import { App, Stack, StackProps } from 'aws-cdk-lib';
 import { Construct } from 'constructs';
 import * as amplify from '../lib';
+import { Asset } from 'aws-cdk-lib/aws-s3-assets';
+import { IntegTest } from '@aws-cdk/integ-tests-alpha';
 
 class TestStack extends Stack {
   constructor(scope: Construct, id: string, props?: StackProps) {
     super(scope, id, props);
 
-    // FIXME: currently this is only released as part of amplify-alpha, which cannot be tested here
-    // and the asset hash is different in alpha
+    // Deploying this integ test would create an Amplify app with branch name 'main'
+    // Re-deploying the stack again after updating the asset file would update the Amplify app.
```

Comment:
```suggestion
    // Deploying this integ test create san Amplify app with branch name 'main'
    // Re-deploying the stack again after updating the asset file updates the Amplify app.
```

Reply:
Can you explain why this is relevant as well?

Reply:
Wanted to do some clean up on Cory's comments for the test doesn't work which was from a while ago. But, removed it.